Resources/doc/features/symfony-http-cache.rst

```diff
@@ -76,6 +76,24 @@ To register subscribers that you need to instantiate yourself, overwrite
         return $subscribers;
     }
 
+.. warning::
+
+    Since Symfony 2.8, the class cache (``classes.php``) is compiled even in console mode 
+    by an optional warmer (``ClassCacheCacheWarmer``). This can produce conflicting results
+    with the regular web entry points, because the class cache may contain definitions
+    (such as the subscribers above) that are autoloaded before the class cache itself; 
+    leading to redeclaration fatal errors.
+    
+    There are two workarounds:
+    
+    * Disable class cache warming in console mode with e.g. a compiler pass::
+    
+        $container->getDefinition('kernel.class_cache.cache_warmer')->clearTag('kernel.cache_warmer');
+        
+    * Force autoloading all classes used by your custom ``HttpCache`` by constructing an
```

Comment:
Force loading of all classes and interfaces used by the ``HttpCache`` in ``app/console`` to have them omitted from the class cache. The simplest way is by calling ``class_exists`` or ``interface_exists`` with them.